apps/parsley/src/hooks/useSections/index.ts

```diff
@@ -111,14 +117,23 @@ export const useSections = ({
   };
 };
 
-const populateSectionState = (sectionData: SectionData): SectionState => {
+const populateSectionState = (
+  sectionData: SectionData,
+  openSectionContainingLine: number | undefined,
```

Comment:
just wondering if this parameter name `openSectionContainingLine` should match the one above `onInitOpenSectionContainingLine`!

Reply:
I think it should be different. populateSectionState is a function that returns a sectionState with default values based on certain inputs. It doesn't really know whether the lineNumber is an initialization value or onChange value. This function may be reused later.